sdk/storage/storage-blob/src/AppendBlobClient.ts

```diff
@@ -385,8 +385,8 @@ export class AppendBlobClient extends BlobClient {
    * Creates a 0-length append blob. Call AppendBlock to append data to an append blob.
    * @see https://docs.microsoft.com/rest/api/storageservices/put-blob
    *
-   * @param {AppendBlobCreateOptions} [options] Options to the Append Block Create operation.
-   * @returns {Promise<AppendBlobsCreateResponse>}
+   * @param {AppendBlobCreateOptions} options Options to the Append Block Create operation.
+   * @returns {Promise<AppendBlobCreateResponse>}
```

Comment:
[options] means options is optional. I don't think you should remove [ and ]

Reply:
@XiaoningLiu Can you elaborate on this comment? We are removing the square brackets around the parameters across the library and other libraries. 

Have you been using the square brackets to indicate that a parameter is optional? Can you point us to any documentation where the square brackets are meant to be used for optional parameters?

Reply:
Never mind, I found something. https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

Reply:
logged #5806

Reply:
Thanks Ramya! I do understand the code freeze date is almost there. But please take some time to update the PR and bring squares back, I don't think there is a solid reason to against JSDoc standard about **optional parameters** and **parameters with default values**. Customers may take optional parameters as reuqired if removing them. Please have an update, then it's good to merge.

Reply:
Updated the PR and brought back the [] that were removed.